hadoop-ozone/dist/src/main/smoketest/debug/auditparser.robot

```diff
@@ -22,16 +22,11 @@ Resource            ../ozone-lib/freon.robot
 Test Timeout        5 minutes
 
 *** Variables ***
-${user}              hadoop
+${user}              testuser/om@EXAMPLE.COM
```

Comment:
The original value allows the test to pass in unsecure environment.  I think we should keep setting `${user}` conditionally.  To avoid problem due to `om1` (hostname) vs. `om` (part of principal), please call `Get test user principal`:

https://github.com/apache/ozone/blob/0e8c8d4d441ec91768f8ab724ce0f29dc657a691/hadoop-ozone/dist/src/main/smoketest/commonlib.robot#L28-L32